packages/paste-website/src/components/Typography.tsx

```diff
@@ -31,8 +31,11 @@ export const InlineCode: React.FC = (props) => {
     <Box
       as="code"
       backgroundColor="colorBackground"
+      borderWidth="borderWidth10"
+      borderColor="colorBorderWeaker"
+      borderStyle="solid"
       borderRadius="borderRadius20"
-      color="colorText"
+      color="inherit"
       fontFamily="fontFamilyCode"
       fontSize="fontSize20"
       lineHeight="lineHeight20"
```

Comment:
The padding here seems like it could use some tweaking. In the avatar page, I see this example of two inline code snippets touching:
![image](https://user-images.githubusercontent.com/77117846/142956421-47b30234-a680-489e-8ab4-676aea7c9005.png)

It looks like the padding in the figma example is also less extreme.

Reply:
Thanks for catching this! I adjusted the padding to account for the extra border pixels and added a story to catch this example.